inference-engine/thirdparty/clDNN/tests/test_cases/convert_color_gpu_test.cpp

```diff
@@ -0,0 +1,529 @@
+
+        throw std::runtime_error(str_message + str_number);
+    }
+}
+
+struct OpenCL {
+    cl::Context _context;
+    cl::Device _device;
+    cl::CommandQueue _queue;
+
+    OpenCL() {
+        // get Intel iGPU OCL device, create context and queue
+        {
+            static constexpr auto INTEL_PLATFORM_VENDOR = "Intel(R) Corporation";
+            const uint32_t device_type = CL_DEVICE_TYPE_GPU;  // only gpu devices
+            const uint32_t device_vendor = 0x8086;  // Intel vendor
+
+            cl_uint n = 0;
+            cl_int err = clGetPlatformIDs(0, NULL, &n);
+            checkStatus(err, "clGetPlatformIDs");
+
+            // Get platform list
+            std::vector<cl_platform_id> platform_ids(n);
+            err = clGetPlatformIDs(n, platform_ids.data(), NULL);
+            checkStatus(err, "clGetPlatformIDs");
+
+            for (auto& id : platform_ids) {
+                cl::Platform platform = cl::Platform(id);
+
+                auto vendor_id = platform.getInfo<CL_PLATFORM_VENDOR>();
+                if (vendor_id != INTEL_PLATFORM_VENDOR)
+                    continue;
+
+                std::vector<cl::Device> devices;
+                platform.getDevices(CL_DEVICE_TYPE_GPU, &devices);
+                for (auto& d : devices) {
+                    if (d.getInfo<CL_DEVICE_TYPE>() == device_type ||
+                        d.getInfo<CL_DEVICE_VENDOR_ID>() == device_vendor) {
+                        _device = d;
+                        _context = cl::Context(_device);
+                        goto greateQueue;
+                    }
+                }
+            }
+            greateQueue:
+            cl_command_queue_properties props = CL_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE;
+            _queue = cl::CommandQueue(_context, _device, props);
+        }
+    }
+    void releaseOclImage(std::shared_ptr<cl_mem> image) {
+        checkStatus(clReleaseMemObject(*image), "clReleaseMemObject");
+    }
+};
+
+std::vector<float> createReferenceData2(std::vector<unsigned char> data, int width, int height, cldnn::format format) {
+    auto img = std::vector<float>(width * height * 3);
+    for (int i = 0; i < height; i++) {
+        for (int j = 0; j < width; j++) {
+            int y_comp = data[i * width + j];
+            int u_comp = data[width * height + i / 2 * width + ((j >> 1) << 1)];
+            int v_comp = data[width * height + i / 2 * width + ((j >> 1) << 1) + 1];
+
+            float B = (1.164f * (float)(y_comp - 16) + 1.596f * (float)(v_comp - 128));
+            float G = (1.164f * (float)(y_comp - 16) - 0.813f * (float)(v_comp - 128) - 0.391f * (u_comp - 128));
+            float R = (1.164f * (float)(y_comp - 16) + 2.018f * (float)(u_comp - 128));
+
+            R = std::min(std::max(R, 0.f), 255.f);
+            G = std::min(std::max(G, 0.f), 255.f);
+            B = std::min(std::max(B, 0.f), 255.f);
+
+            if (format == cldnn::format::bfyx) {
```

Comment:
Is it really needed? I don't see support of bfyx into the implementation_map related to the primitive.